soh/src/overlays/actors/ovl_En_Box/z_en_box.c

```diff
@@ -569,7 +569,7 @@ void EnBox_SpawnIceSmoke(EnBox* this, PlayState* play) {
     if (Rand_ZeroOne() < 0.3f) {
         f0 = 2.0f * Rand_ZeroOne() - 1.0f;
         pos = this->dyna.actor.world.pos;
-        if (this->type == ENBOX_TYPE_SMALL || this->type == ENBOX_TYPE_6 || this->type == ENBOX_TYPE_ROOM_CLEAR_SMALL ||
+        if (this->type == ENBOX_TYPE_SMALL || this->type == ENBOX_TYPE_6 || this->type == ENBOX_TYPE_ROOM_CLEAR_SMALL || 
```

Comment:
LOL, I love random whitespace changes. Thanks VS.

Reply:
It's the added ` ` after the `||` at the end of the line, not quite random

Reply:
i absolutely despise that about VS, inflates PRs for no reason.

Reply:
> It's the added ` ` after the `||` at the end of the line, not quite random

It's random in that VS decides to add things like it at random times. Didn't always seem to be logic to the timing.